### PR TITLE
Typo fixes for issue #6487

### LINKED ISSUE #6487: Minor typo fixes in the 'Authorization for groups and list claims' task under Security
https://preliminary.istio.io/docs/tasks/security/authorization/rbac-groups/

### PR DESCRIPTION
**Pull request by adammil2000**
Fixes #6487

### CODE REVIEW

**Comment by adammil2000** [comment]
#6487

File: content/en/docs/tasks/security/authorization/rbac-groups/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Authorization for groups and list claims
-description: Tutorial on how to configure the groups-base authorization and configure the authorization of list-typed claims in Istio.
+description: Tutorial on how to configure the groups-based authorization and configure the authorization of list-typed claims in Istio.
```
Comment (elysdir) [inline content/en/docs/tasks/security/authorization/rbac-groups/index.md]: Looks like this same typo appears in the opening line of the page (https://deploy-preview-6526--preliminary-istio.netlify.com/docs/tasks/security/authorization/rbac-groups/): "This tutorial walks you through examples to configure the groups-base authorization..." Do you want to fix that as well, while you're here?

File: content/en/docs/tasks/security/authorization/rbac-groups/index.md [hunk 2]
```diff
@@ -168,6 +168,8 @@ wait until the newly defined authorization policy to take effect.
     EOF
     {{< /text >}}
 
+    Wait for the newly defined policy to take effect.
```
Comment (elysdir) [inline content/en/docs/tasks/security/authorization/rbac-groups/index.md]: Would it be a good idea to say slightly more about roughly how long they should expect to wait? (For example, is it more like seconds or more like weeks?) Fine to leave alone, of course.

Comment (adammil2000) [reply content/en/docs/tasks/security/authorization/rbac-groups/index.md]: Fixed

File: content/en/docs/tasks/security/authorization/rbac-groups/index.md (hunk 1 above)
Comment (adammil2000) [reply content/en/docs/tasks/security/authorization/rbac-groups/index.md]: Fixed

**Pull request merged**